<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>139</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>학번</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>139</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_text_edit2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pw_text_edit2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>complete_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>home()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>complete_login()</slot>
  <slot>home()</slot>
 </slots>
</ui>
